CONTRIBUTING.md

```diff
@@ -61,12 +57,28 @@ However, when speaking about those or explaining we use `actor` noun form: `stor
 * Documentation
 * Code comments
 
-## Pull Request Process
+## Development
+
+Following section explains various suggestions and procedures while development of Thanos.
+
+### Prerequisites
+
+* It is strongly recommended that you use OSX or popular Linux distributions systems e.g. Ubuntu, Redhat, or OpenSUSE for development.
+* Go SDK 1.13.9 or newer installed.
+
+### First steps
+
+It's a key to get familiar with style guides and mechanics of Thanos, especially if your contribution touches more than on component of the Thanos distributed system.
+We recommend:
+
+* Reading [getting started docs](docs/getting-started.md) and getting through this. Alternatively https://katacoda.com/thanos.
+* Familiarize yourself with our [coding style guides.](docs/contributing/coding-style-guide.md).
```

Comment:
grammar, to keep this list correct, we need the start of each item to end in `-ing`. Otherwise one is an action while another is an instruction.

```suggestion
* Familiarizing yourself with our [coding style guidelines.](docs/contributing/coding-style-guide.md).
```